docs/contents/dev/dev-write-1st-app.md

```diff
@@ -9,7 +9,7 @@ We'll use the classical [wordcount](https://github.com/apache/incubator-gearpump
 	  override val options: Array[(String, CLIOption[Any])] = Array.empty
 	
 	  override def main(akkaConf: Config, args: Array[String]): Unit = {
-	    val context = ClientContext(akkaConf)
+	    val context = RuntimeEnvironment.get().newClientContext(akkaConf)
```

Comment:
Can we hide `RuntimeEnvironment.get()` in `ClientContext` since users don't need to know ? Another benefit is there will be no changes to existing applications.